testing/nss_wrapper/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor: Terror <miguel.a.terron@gmail.com>
+# Maintainer: Terror <miguel.a.terron@gmail.com>
+pkgname=nss_wrapper
+pkgver=1.1.5
+pkgrel=0
+pkgdesc="A wrapper for the user, group and hosts NSS API"
+url="https://cwrap.org/nss_wrapper.html"
+arch="all"
+license="BSD-3-Clause"
+depends=""
+makedepends="cmake cmocka-dev"
+install=""
+subpackages="$pkgname-doc"
+source="https://ftp.samba.org/pub/cwrap/$pkgname-$pkgver.tar.gz nss.h"
```

Comment:
I would split nss.h onto a new line making it easier to spot.
```
source="https://ftp.samba.org/pub/cwrap/$pkgname-$pkgver.tar.gz
        nss.h
        "
```

Reply:
Will do.

Reply:
Would it be better if I close this PR and submit a new one?